packages/web/pages/matches/character/[characterName].tsx

```diff
@@ -0,0 +1,75 @@
+import { CombatStubList } from '@wowarenalogs/shared';
+import { LocalRemoteHybridCombat } from '@wowarenalogs/shared/src/components/CombatStubList/rows';
+import { LoadingPage } from '@wowarenalogs/shared/src/components/common/LoadingPage';
+import { QuerryError } from '@wowarenalogs/shared/src/components/common/QueryError';
+import { useGetCharacterMatchesLazyQuery } from '@wowarenalogs/shared/src/graphql/__generated__/graphql';
+import _ from 'lodash';
+import { useRouter } from 'next/router';
+import { useEffect } from 'react';
+import { TbLoader, TbRocketOff } from 'react-icons/tb';
+
+const Page = () => {
+  const router = useRouter();
+  const { realm, characterName } = router.query;
```

Comment:
an alternative is to support character names formatted as "name-realm" which is the standard by blizzard I think. That would allow easier copy/pasting into our URL. What do you think?

Reply:
we could eventually do a {char}-{realm} style but we'd need to make sure a bunch of normalization happened before doing the query